package.json

```diff
@@ -34,36 +34,44 @@
     "locales": "./locales/"
   },
   "scripts": {
-    "build": "make build",
+    "build": "rollup -c",
```

Comment:
Probably should change this back to `make`. My bad.

Reply:
Sounds good. Done.